tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/ArgoCdConstants.java

```diff
@@ -36,4 +36,13 @@ public class ArgoCdConstants {
     }});
 
     static final String ARGOCD_NAMESPACE = "argocd";
+
+    static final List<String> DEFAULT_SOURCE_REPOS = Collections.singletonList("*");
+
+    static final List<Map<String, String>> DEFAULT_DESTINATIONS = Collections.singletonList(new HashMap<String, String>()
+    {{
+        put("name", "all");
+        put("server", "*");
+        put("namespace", "*");
+    }});
```

Comment:
Double-brace initialization is discouraged du to potential for memory leaks and a new anonymous class every time it's used. You can use a static method to initialize or use a `static` block like:
```java
    static final List<Map<String, String>> DEFAULT_DESTINATIONS;

    static {
        Map<String, String> m = new HashMap<>();
        m.put("name", "all");
        m.put("server", "*");
        m.put("namespace", "*");
        DEFAULT_DESTINATIONS = Collections.singletonList(m);
    }
```

Reply:
Updated